### PR TITLE
feat: add 'skaffold inspect jobManifestPath' and 'skaffold transform-schema jobManifestPath' commands

### LINKED ISSUE #8573: Add abillity to modify `skaffold verify` jobManifestPath
Issue tracks adding the abillity to modify `skaffold verify` jobManifestPath by adding new commands to support this use case

### PR DESCRIPTION
**Pull request by aaron-prindle**
fixes #8573

### CODE REVIEW

File: pkg/skaffold/inspect/jobManifestPaths/modify.go [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+var (
+	getCfgs = parser.GetAllConfigs
+)
+
+func Modify(ctx context.Context, out io.Writer, opts config.SkaffoldOptions, inputFile, outputFile string) error {
+	// Open our jsonFile
+	jsonFile, err := os.Open(inputFile)
+	if err != nil {
+		fmt.Println(err)
```
Comment (renzodavid9) [inline pkg/skaffold/inspect/jobManifestPaths/modify.go:42]: Shouldn't we return the err here? Or log it with `log.Entry(context.TODO()).Error`?

File: pkg/skaffold/inspect/jobManifestPaths/modify.go [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+	}
+	// defer the closing of our jsonFile so that we can parse it later on
+	defer jsonFile.Close()
+
+	byteValue, _ := ioutil.ReadAll(jsonFile)
+
+	var result jobManifestPathList
+	json.Unmarshal(byteValue, &result)
```
Comment (renzodavid9) [inline pkg/skaffold/inspect/jobManifestPaths/modify.go:50]: This operation can return an error, if the `inputFile` is not a json file. I noticed that we fail silently, and the generated output is the same skaffold.yaml without the modifications. Should we also log the error from the unmarshal?

File: pkg/skaffold/inspect/jobManifestPaths/modify.go (hunk 1 above)
Comment (aaron-prindle) [reply pkg/skaffold/inspect/jobManifestPaths/modify.go]: Yep, good catch.  This now properly handles errors and has a test case related to this

File: pkg/skaffold/inspect/jobManifestPaths/modify.go (hunk 2 above)
Comment (aaron-prindle) [reply pkg/skaffold/inspect/jobManifestPaths/modify.go]: Yep, good catch.  This now properly handles errors and has a test case related to this

File: pkg/skaffold/inspect/jobManifestPaths/modify.go (hunk 1 above)
Comment (aaron-prindle) [reply pkg/skaffold/inspect/jobManifestPaths/modify.go]: Also logs the error as well to stdout for users to know what happened

File: pkg/skaffold/inspect/jobManifestPaths/modify.go (hunk 2 above)
Comment (aaron-prindle) [reply pkg/skaffold/inspect/jobManifestPaths/modify.go]: Also logs the error as well to stdout for users to know what happened

**Review by renzodavid9 — APPROVED** [review]

**Pull request merged**